### ISSUE TITLE
Screen Read Close Button on Pop Up

### ISSUE THREAD

**Issue by fongcindy** [GSA/px-benefit-finder#817]
At the Select an option pop-up, close button on the pop-up, the screen reader announces "button" for the X. We expect to hear "Close button" when the focus is on the X of the modal. 

https://app.screencast.com/fOkxQANcTnG4Q?conversation=5mQynAYFFdKQQBNogsuUZW

**Comment by scottqueen-bixal** [comment]
@fongcindy there is no hint for the DOB field since there is no content provided.

<img width="1440" alt="Screenshot 2024-01-24 at 11 25 21 AM" src="https://github.com/GSA/px-benefit-finder/assets/37077057/1717d946-1967-41dc-ab4b-bee7ca71e190">

Should be addressable by including the proper hint content in the CMS.

**Comment by scottqueen-bixal** [comment]
@angelina-smith this is on sandbox for review https://bf-cms-main.bxdev.net/benefit-finder/death

**Comment by fongcindy** [comment]
@scottqueen-bixal Angelina is unable to use her screen reader software in VDI, is there anyway to push this (and [810 ](https://github.com/GSA/px-benefit-finder/issues/810)& [818](https://github.com/GSA/px-benefit-finder/issues/818)) into production and have Angelina QA when it is live?

**Comment by scottqueen-bixal** [comment]
@fongcindy and @angelina-smith, I don't think that is the approach we would want to take. 

It would take a few weeks to do that and the idea is to test these before release,

In the past we have been using storybook for QA items that required the screen reader tools. Is there any reason why we can't continue to do that?

Use the App Component: https://benefit-finder-storybook-main.app.cloud.gov/?path=/story/app--primary
Full Screen View: https://benefit-finder-storybook-main.app.cloud.gov/iframe.html?args=&id=app--primary&viewMode=story

**Comment by angelina-smith** [comment]
@scottqueen-bixal Experiencing the same screen reader issue on this Close button as in #818 . Screen reader is saying "Close button button".

[Screen recording of "Close button button"](https://app.screencastify.com/v3/watch/3TgskkE1Bg9TCcekuHTY). And again, if the link to the screen recording isn't working. Would love to hop on a quick call and share my screen

**Comment by angelina-smith** [comment]
@scottqueen-bixal It sounds great! The "Continue" button on the 'About the deceased' page and the "Close" button on the pop-up sound like how we would want it!
